db/dbtypes/types.go

```diff
@@ -988,32 +987,25 @@ func MergeRows(rows []*AddressRow) ([]AddressRowMerged, error) {
 		}
 	}
 
-	// Build the slice.
-	mergedRows := make([]AddressRowMerged, 0, len(merged))
-	for i := range hashes {
-		mergedRows = append(mergedRows, *merged[hashes[i]])
```

Comment:
We eliminate this final loop over the map, but as a consequence, we cannot accurately pre-allocate the `mergedRows` slice.